dbt_copilot_helper/commands/copilot.py

```diff
@@ -100,10 +97,11 @@ def _normalise_keys(source: dict):
     if not config:
         return {}
 
-    try:
-        validate_json(instance=config, schema=schema)
-    except jsonschema.exceptions.ValidationError as err:
-        click.echo(click.style(err, fg="red"))
+    errors = validate_addons(config)
+    if errors:
+        click.echo(click.style(f"Errors found in {config_file}:", fg="red"))
+        for addon, error in errors.items():
```

Comment:
might be worth adding / updating a test for this new logic for outputting multiple errors